{"name": "test_login_error[alex' -- x-abc123-用户名和密码不匹配]", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'find_elment'", "trace": "self = <测试.iwebshop.test_scripts.test_login.TestLogin object at 0x000002461E945470>, user = \"alex' -- x\", pwd = 'abc123', exp = '用户名和密码不匹配'\n\n    @pytest.mark.parametrize(\"user,pwd,exp\", get_param(\"login_error\"))\n    def test_login_error(self, user, pwd, exp):\n        self.login_obj.login(user, pwd)\n>       assert exp in self.login_obj.get_not_match()\n\ntest_scripts\\test_login.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <page.login_page.Login object at 0x000002461E896BE0>\n\n    def get_not_match(self):\n>       return self.find_elment(settings.EL_NOT_MATCH)\nE       AttributeError: 'Login' object has no attribute 'find_elment'\n\npage\\login_page.py:26: AttributeError"}, "parameters": [{"name": "user", "value": "'alex' -- x'"}, {"name": "pwd", "value": "'abc123'"}, {"name": "exp", "value": "'用户名和密码不匹配'"}], "start": 1573743819458, "stop": 1573743819867, "uuid": "acb838db-d4ba-400e-8604-f16af780712c", "historyId": "e226953c7cc811dfdfe51d340ee4a7ff", "testCaseId": "6ce952dc078fe00d58c03ef3f9899bc3", "fullName": "test_scripts.test_login.TestLogin#test_login_error", "labels": [{"name": "parentSuite", "value": "test_scripts"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-JKISR1H"}, {"name": "thread", "value": "45488-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_scripts.test_login"}]}